src/test/java/racingcar/domain/NameTest.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.domain;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class NameTest {
+    @DisplayName("자동차 이름이 1자 미만, 5자 초과하여 유효하지 않은 경우")
```

Comment:
```suggestion
    @DisplayName("자동차 이름이 1자 미만, 5자 초과하여 유효하지 않은 경우 예외가 발생한다")
```

이정도로 결과까지 명확하게 서술하면 더 좋습니다.